packages/uxpin-merge-cli/bin/integrate-package

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+
+const program = require('commander');
+const cp = require('child_process');
+const path = require('path');
+const fs = require('fs-extra');
+
+program
+    .option('--token <string>' , 'Library token')
+    .option('--package-name <string>', 'NPM Package name')
+    .option('--components <string>', 'List of imported components and categories')
+    .option('--styles <url>', 'Url to CSS Styles');
+
+program.parse(process.argv);
+
+const uxpinMergePath = path.resolve(__dirname, './uxpin-merge');
+const appPath = path.resolve(process.cwd(), program.token);
+
+try {
+    const creatingAppResult = cp.spawnSync(uxpinMergePath, [
```

Comment:
I've tested this and working great! Can we print the stdout and stderr from the command? :)

Reply:
sure